Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Processing/ImageAssetProducer.cpp

```diff
@@ -197,9 +197,7 @@ namespace ImageProcessingAtom
             // Add all the mip chain assets as dependencies except the tail mip chain since its embedded in the StreamingImageAsset
             if (it->Get() != mipChains.begin()->Get())
             {
-                // Use PreLoad for mipchain assets for now.
-                // [GFX TODO] [ATOM-14467] Remove unnecessary code in StreamingImage::OnAssetReloaded when runtime switching dependency load behavior is supported
-                product.m_dependencies.push_back(AssetBuilderSDK::ProductDependency(it->GetId(), AZ::Data::ProductDependencyInfo::CreateFlags(AZ::Data::AssetLoadBehavior::PreLoad)));
+                product.m_dependencies.push_back(AssetBuilderSDK::ProductDependency(it->GetId(), AZ::Data::ProductDependencyInfo::CreateFlags(AZ::Data::AssetLoadBehavior::NoLoad)));
```

Comment:
Maybe add a comment to say why we dont PreLoad in case someone changes it in future.